src/components/Input/Textarea.tsx

```diff
@@ -7,7 +7,7 @@ import * as Rivet from '../util/Rivet';
 import { renderInput, TextProps } from './common';
 
 const Textarea :  React.SFC<TextProps & React.TextareaHTMLAttributes<HTMLTextAreaElement>> = 
-    renderInput((attrs) => <textarea {...attrs}/>);
+    renderInput("textarea");
```

Comment:
This should use single quotes to be consistent

```suggestion
    renderInput('textarea');
```